R/logistic_mle.R

```diff
@@ -65,3 +63,17 @@ logistic.mle.newton <-
 
     matrix(coef, ncol = 1)
   }
+
+#'
+take.one.newton.step <- function(design,
+                                 outcome,
+                                 coef, solver) {
+  hessian <-
+    logistic.log.likelihood.hessian(coef, design, outcome)
+  gradient <-
+    logistic.log.likelihood.gradient(coef, design, outcome)
+
+  delta_coef <- -solve.linear.system(hessian, gradient, solver)
+
+  return(coef + delta_coef)
```

Comment:
```suggestion
  delta_coef <- solve.linear.system(hessian, gradient, solver)

  return(coef - delta_coef)
```

super minor, but (at least in my experience), newton steps are usually `coef - delta` as opposed to `coef+delta`, and this saves a redundant operation